daprdocs/content/en/operations/components/setup-pubsub/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -5,51 +5,63 @@ linkTitle: "Azure Events Hub"
-
-The next step is to create a Dapr component for Azure Event Hubs.
-
-Create the following YAML file named `eventhubs.yaml`:
+## Format
 
 ```yaml
 apiVersion: dapr.io/v1alpha1
 kind: Component
 metadata:
-  name: <NAME>
-  namespace: <NAMESPACE>
+  name: eventhubs-pubsub
+  namespace: default
 spec:
   type: pubsub.azure.eventhubs
   version: v1
   metadata:
   - name: connectionString
-    value: <REPLACE-WITH-CONNECTION-STRING> # Required. "Endpoint=sb://****"
+    value: "Endpoint=sb://{EventHubNamespace}.servicebus.windows.net/;SharedAccessKeyName={PolicyName};SharedAccessKey={Key};EntityPath={EventHub}"
   - name: storageAccountName
-    value: <REPLACE-WITH-STORAGE-ACCOUNT-NAME> # Required.
+    value: "myeventhubstorage"
   - name: storageAccountKey
-    value: <REPLACE-WITH-STORAGE-ACCOUNT-KEY> # Required.
+    value: "112233445566778899"
   - name: storageContainerName
-    value: <REPLACE-WITH-CONTAINER-NAME > # Required.
+    value: "myeventhubstoragecontainer"
 ```
 
-See [here](https://docs.microsoft.com/en-us/azure/event-hubs/authorize-access-shared-access-signature) on how to get the Event Hubs connection string. Note this is not the Event Hubs namespace.
-
 {{% alert title="Warning" color="warning" %}}
 The above example uses secrets as plain strings. It is recommended to use a secret store for the secrets as described [here]({{< ref component-secrets.md >}}).
 {{% /alert %}}
 
+## Spec metadata fields
+
+| Field              | Required | Details | Example |
+|--------------------|:--------:|---------|---------|
+| connectionString    | Y  | Connection-string for the Event Hubs  | `Endpoint=sb://{EventHubNamespace}.servicebus.windows.net/;SharedAccessKeyName={PolicyName};SharedAccessKey={Key};EntityPath={EventHub}`
+| storageAccountName  | Y  | The storage account name to use for the EventProcessorHost   |`"myeventhubstorage"`
+| storageAccountKey   | Y  | Storage account key  to use for the EventProcessorHost   | `"112233445566778899"`
+| storageContainerName | Y | The storage container name for the storage account name.  | `"myeventhubstoragecontainer"`
+
+## Create a Dapr component
+
+To setup Azure Event Hubs, you create a component of type `pubsub.azure.eventhubs`
+
+## Setup Azure Event Hubs
+
+Follow the instructions [here](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) on setting up Azure Event Hubs.
+Since this implementation uses the Event Processor Host, you will also need an [Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal). Follow the instructions [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage) to manage the storage account access keys. 
+
+See [here](https://docs.microsoft.com/en-us/azure/event-hubs/authorize-access-shared-access-signature) on how to get the Event Hubs connection string. Note this is not the Event Hubs namespace.
+
 ## Create consumer groups for each subscriber
```

Comment:
I'd make this a ### header instead of ## so that readers understand that it's part of creating the Event Hubs instance